### DISCUSSION TITLE
<title>Help

### DISCUSSION THREAD

**Discussion by mrdiep999** [community/community#65906] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

how to sync obsidian through github?

**Comment by Linh777GKOG** [comment]
1. Create a GitHub Repository:
2. Clone Your GitHub Repository:
3. Move or Create Your Obsidian Vault:
You can either create a new Obsidian vault within the cloned repository directory or move your existing vault into it.

**Comment by Linh777GKOG** [comment]
Remember to commit and push regularly to keep your Obsidian notes synced with GitHub. You can also use a Git GUI client for a more user-friendly experience if you prefer.
